sycl/include/CL/sycl/queue.hpp

```diff
@@ -286,11 +286,11 @@ class queue {
   /// Provides additional information to the underlying runtime about how
   /// different allocations are used.
   ///
-  /// \param Ptr is a USM pointer to the allocation.
-  /// \param Length is a number of bytes in the allocation.
-  /// \param Advice is a device-defined advice for the specified allocation.
-  /// \return an event representing advice operation.
-  event mem_advise(const void *Ptr, size_t Length, int Advice);
+  /// @param Ptr is a USM pointer to the allocation.
+  /// @param Length is a number of bytes in the allocation.
+  /// @param Advice is a device-defined advice for the specified allocation.
+  /// @return an event representing advice operation.
```

Comment:
Please, revert these changes.

Reply:
@bader pi_mem_advice is introduced in this patch. Reverting this in this file will require removing it altogether. Is that what we want?

Reply:
This request is about comments only.

Reply:
that makes sense. on it.